services/scan/src/store.ts

```diff
@@ -864,6 +865,29 @@ export class Store {
       const backAdjudications = backAdjudicationJson
         ? safeParseJson(backAdjudicationJson, MarkAdjudicationsSchema).ok()
         : undefined;
+
+      const frontImage: InlineBallotImage = { normalized: '' };
+      const backImage: InlineBallotImage = { normalized: '' };
```

Comment:
Do we want these to be set with an invalid value if there is no image? Or just leave them out?